miprometheus/problems/seq_to_seq/seq_to_seq_problem.py

```diff
@@ -20,10 +20,12 @@
 
 """
 
-__author__ = "Tomasz Kornuta & Vincent Marois"
+__author__ = "Tomasz Kornuta & Vincent Marois & Vincent Albouy"
 
 from miprometheus.problems.problem import Problem
 import torch
+from miprometheus.utils.app_state import AppState
```

Comment:
No need because Problem already has 
https://github.com/IBM/mi-prometheus/blob/68ad04da795c2d5e8ecb86144cb459a28ac1aa71/miprometheus/problems/problem.py#L136

Reply:
ok but some classes like TextToTextProblem do not see it somehow